### DISCUSSION TITLE
py file

### DISCUSSION THREAD

**Discussion by guiworldtv2** [streamlink/streamlink#4849] [Q&A]
Hello friends
I'm building a schema in python notebook and I want to know what is the master .py file of streamlink?
some cases I need a plugin for youtube, other cases I need other plugins to work and it is difficult to test all plugins to know what is correct

do you have any python .py have all the schema with access to the streamlink module?

**Comment by gravyboat** [comment]
I don't understand what you're asking that isn't answered by looking at `src/streamlink/__main__.py` or `src/streamlink_cli/main.py` and reading the code from there. We don't have the bandwidth and can't support you in writing your third party tool or application.
